### PR TITLE
Docs for media service

### PR DESCRIPTION
**Pull request by legendofa**

### CODE REVIEW

File: docs/docs/dev-manuals/services/media.mdx [hunk 1]
```diff
@@ -0,0 +1,87 @@
+import CustomImage from "@site/src/components/CustomImage";
+
+# Media Service
+
+The media service provides the bounded context `Media`.
+It is responsible for managing medias.
```
Comment (delvh) [inline docs/docs/dev-manuals/services/media.mdx]:
```suggestion
It is responsible for managing assets.
```

File: docs/docs/dev-manuals/services/media.mdx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+The media services uses a Nginx reverse proxy to make the fixed domain of MinIO pre-signed URLs configurable through environment variables.
+
+The service parses the following environment variables:
+
+- `MINIO_ENDPOINT`: configurable domain of the MinIO instance.
+- `PATH_EXPIRATION_TIME`: validity duration of MinIO pre-signed URLs in seconds.
```
Comment (delvh) [inline docs/docs/dev-manuals/services/media.mdx]: What happens when the duration is exceeded?
We should probably elaborate on that…

Comment (delvh) [reply docs/docs/dev-manuals/services/media.mdx]: Also, it is probably a good idea to elaborate on what a presigned URL is, what it does, and what it means for users and developers.

File: docs/docs/dev-manuals/services/media.mdx (hunk 1 above)
Comment (legendofa) [reply docs/docs/dev-manuals/services/media.mdx]: Would it be ok to change it to "media files"? Assets could be confused with financial assets

Comment (delvh) [reply docs/docs/dev-manuals/services/media.mdx]: Sounds fine

File: docs/docs/dev-manuals/services/media.mdx [hunk 3]
```diff
@@ -3,18 +3,23 @@ import CustomImage from "@site/src/components/CustomImage";
 # Media Service
 
 The media service provides the bounded context `Media`.
-It is responsible for managing medias.
+It is responsible for managing media files.
```
Comment (delvh) [inline docs/docs/dev-manuals/services/media.mdx]:
```suggestion
It is responsible for managing media files such as images.
```

File: docs/docs/dev-manuals/services/media.mdx [hunk 4]
```diff
@@ -3,18 +3,23 @@ import CustomImage from "@site/src/components/CustomImage";
...
 The media services uses a Nginx reverse proxy to make the fixed domain of MinIO pre-signed URLs configurable through environment variables.
 
+A pre-signed URL is used as a reference to a specific media. It is generated for each `Media` entity when the `path` attribute is requested.
+This is necessary for access control to specific media files. A `Media` entity cannot be queried by a user client directly.
+It is used in within the service mesh by other services to reference medias in the services datatypes.
+Pre-signed URLs link media to authenticated service datatypes, chaining authentication. Each request of the `path` attribute on `Media` generates a new pre-signed URL with a limited validity duration.
```
Comment (delvh) [inline docs/docs/dev-manuals/services/media.mdx:12]: > So, in other words the presigned URLS are generated automatically without my interference?
And an attachment will always vanish once the presigned URL is no longer valid?

Users reading this, probably

Comment (legendofa) [reply docs/docs/dev-manuals/services/media.mdx:12]: > > So, in other words the presigned URLS are generated automatically without my interference?
> > And an attachment will always vanish once the presigned URL is no longer valid?
> 
> Users reading this, probably

Yes, exactly

File: docs/docs/dev-manuals/services/media.mdx [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+# Media Service
+
+The media service provides the bounded context `Media`.
+It is responsible for managing media files such as images.
+The media services uses a Nginx reverse proxy to make the fixed domain of MinIO pre-signed URLs configurable through environment variables.
+
+A pre-signed URL is used as a reference to a specific media. It is generated for each `Media` entity when the `path` attribute is requested.
+This is necessary for access control to specific media files. A `Media` entity cannot be queried by a user client directly.
+It is used in within the service mesh by other services to reference medias in the services datatypes.
+Pre-signed URLs link media to authenticated service datatypes, chaining authentication. Each request of the `path` attribute on `Media` generates a new pre-signed URL with a limited validity duration.
```
Comment (delvh) [inline docs/docs/dev-manuals/services/media.mdx:12]:
```suggestion
Pre-signed URLs link media to authenticated service datatypes, chaining authentication. Each request of the `path` attribute on `Media` generates a new pre-signed URL with a limited validity duration.
In other words, presigned URLS are generated automatically without any interference and attachments will always vanish once the presigned URL is no longer valid
```

**Review by delvh — APPROVED** [review]

**Pull request merged**